src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/TargetsGetHandlerTest.java

```diff
@@ -90,7 +90,12 @@ void shouldHandleCorrectPath() {
 
     @Test
     void shouldReturnListOfTargets() {
-        ServiceRef target = new ServiceRef("foo", 1);
+        ServiceRef target;
+        try {
+            target = new ServiceRef("foo", 1, "foo");
+        } catch (Exception e) {
+            target = null;
```

Comment:
Same here, just add a throws-declaration to the method and let it fail right away if somehow the test ServiceRef construction fails.